wifi-subsys/components/uart/src/at_handler.c

```diff
@@ -0,0 +1,213 @@
+#include <stdio.h>
+#include <string.h>
+
+#include "driver/gpio.h"
+#include "driver/uart.h"
+#include "esp_log.h"
+#include "freertos/FreeRTOS.h"
+#include "freertos/task.h"
+#include "subsys_uart.h"
+
+#include "at_handler.h"
+#include "subsys_uart.h"
+#include "wifi.h"
+
+#define ARRAY_SIZE 11
+
+char *default_keys[ARRAY_SIZE] = {
+    "NVS_RST", "WIFI_OFF",  "WAP_SSID", "WAP_PASS",  "WAP_CHAN",  "WAP_AUTH",
+    "WIFI_ON", "WIFI_MODE", "WIFI_RST", "WSTA_SSID", "WSTA_PASS",
+};
+
+char *symbols[4] = {
+    "+",
+    "&",
+    "?",
+    "=",
+};
```

Comment:
Looks that you are using only a character, I suggest use a only array of chars, It's not necessary use an string array.

Reply:
```suggestion
char symbols[4] = {
    '+',
    '&',
    '?',
    '=',
};
```

Reply:
happens that strtok is very sensitive bro,  with a simple char i have memory overflow

Reply:
Oh, sorry man, i didn't see that